builder/relinker/relinker.py

```
@@ -0,0 +1,582 @@
+    secs.sort()
+    return secs
+
+def func2sect(func):
+    if ' ' in func:
+        func_l = func.split(' ')
+    else:
+        func_l = list()
+        func_l.append(func)
+    
+    secs = list()
+    for l in func_l:
```

Ruff (0.15.7)

[error] Ambiguous variable name: `l`

(E741)